File: src/me/krickl/memebotj/InternalCommands/AdminCommands/DebugCommand.scala
package me.krickl.memebotj.InternalCommands.AdminCommands

class DebugCommand(channel: String, command: String = "null", dbprefix: String = "") extends me.krickl.memebotj.CommandHandler(channel, command, dbprefix) {
  this.setNeededCommandPower(75)


  override def commandScript(sender: me.krickl.memebotj.UserHandler, channelHandler: me.krickl.memebotj.ChannelHandler, data: Array[String]) = {
    channelHandler.sendMessage("Congraturations. This glitch is happy end and you have prooven the justive of our coding! You are great debugger !!!", this.getChannelOrigin)
  }
}

File: src/main/java/me/krickl/memebotj/Commands/Internal/DebugCommand.java
package me.krickl.memebotj.Commands.Internal;

import me.krickl.memebotj.ChannelHandler;
import me.krickl.memebotj.Commands.CommandHandler;
import me.krickl.memebotj.UserHandler;

/**
 * This file is part of memebotj.
 * Created by unlink on 11/04/16.
 */
public class DebugCommand extends CommandHandler {
    public DebugCommand(ChannelHandler channelHandler, String commandName, String dbprefix) {
        super(channelHandler, commandName, dbprefix);
    }

    @Override
    public void overrideDB() {
        this.setHelptext("");
        this.setNeededCommandPower(75);
        this.setUserCooldownLength(0);
    }

    @Override
    public void commandScript(UserHandler sender, String[] data) {
    }
}
